pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/SimpleFilter.java

```diff
@@ -49,22 +54,30 @@
  *     @Property(name="service.ranking", intValue=1)
  * })
  */
-//@SlingFilter(order=1, description="A Simple Filter")
-//@Property(name="service.vendor", value="The Apache Software Foundation")
```

Comment:
Lets add some properties here, similar to what the other servlets are using.

Reply:
done